### DISCUSSION TITLE
Feature Request: Clean Download History

### DISCUSSION THREAD

**Discussion by heinz3** [Media-Fetch-Pro/Media-Fetch-Pro#10] [Ideas]
Hello, thank you for this great project!
Maybe this could be an useful extra feature: 
To give the user the ability to delete the history of downloaded files.
By adding an extra button on the tab "History"->"Finished" with button title "Clear download history" .

**Comment by CorrectRoadH** [comment]
Thanks for your feedback.🥺 But I'm busy working on finals from school right now, so I haven't had time to implement it yet. I might do it later.
